docs/reference/cli-commands/start.md

```diff
@@ -22,7 +22,12 @@ any other services it depends on, in the correct order.
 
 ## How it works
 
-When starting a service, Pebble executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly. Assuming the command doesn't exit within that time window, the start is considered successful, otherwise `pebble start` will exit with an error, regardless of the `on-failure` value.
+When starting a service, Pebble executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly.
+
+- If the command doesn't exit within the 1 second window, the start is considered successful.
+- If the command exits within the 1 second window, Pebble retries the command after a backoff delay of half a second (default value). Within the 1 second window, Pebble applies the restart logic described in [Service auto-restart](/reference/service-auto-restart.md). You can configure the backoff delay and the auto-restart behaviour according to the service's exit code. For more information, see [Layer specification](/reference/layer-specification.md).
```

Comment:
I think we should avoid mentioning details of the backoff times ("half a second"), and just link to the "Service auto-restart" doc for that level of detail. I think we can remove much of this paragraph, given it's described there (including config fields):

```suggestion
- If the command exits within the 1 second window, Pebble retries the command after a configurable backoff, using the restart logic described in [Service auto-restart](/reference/service-auto-restart.md).
```

Reply:
+1